Streetcode/Streetcode.BLL/MediatR/Streetcode/Streetcode/Create/CreateStreetcodeHandler.cs

```diff
@@ -151,9 +157,22 @@ await _repositoryWrapper.StreetcodeImageRepository.CreateRangeAsync(imagesIds.Se
 
     private async Task AddImagesDetails(IEnumerable<ImageDetailsDto>? imageDetails)
     {
+        if(imageDetails.IsNullOrEmpty())
+        {
+            throw new HttpRequestException("There is no valid imagesDetails value", null, System.Net.HttpStatusCode.BadRequest);
+        }
+
         if(imageDetails == null)
         {
-            return;
+            throw new HttpRequestException("There is no valid imagesDetails value", null, System.Net.HttpStatusCode.BadRequest);
+        }
+
+        foreach (var detail in imageDetails)
+        {
+            if(detail.Alt == null)
+            {
+                throw new HttpRequestException("There is no valid imagesDetails value", null, System.Net.HttpStatusCode.BadRequest);
+            }
```

Comment:
Assuming that the 'Alt' property is a string, why would we use detail.Alt == null, when we should use detail.Alt.isNullOrEmpty? Can 'Alt' property be an empty string?